fhir-server/src/main/java/org/linuxforhealth/fhir/server/util/RestAuditLogger.java

```diff
@@ -746,6 +746,64 @@ protected static AuditLogEntry populateAuditLogEntry(AuditLogEntry entry, HttpSe
         return entry;
     }
 
+    /**
+     * Populates the passed audit log entry for Bundle entry.
+     * This method will populate the resource id, resource type and version id from the
+     * Bundle response entry location when the return preference is "OperationOutcome".
+     * When the return preference is "representation" the populateAuditLogEntry method will
+     * populate the resource id, resource type and version id.
+     * @param entry
+     *  The AuditLogEntry to be populated.
+     * @param responseEntry
+     *  The Bundle response entry.
+     * @param request
+     *  The HttpServletRequest representation of the REST request.
+     * @param resource
+     *  The Resource object.
+     * @param startTime
+     *  The start time of the request execution.
+     * @param endTime
+     *  The end time of the request execution.
+     * @param responseStatus
+     *  The response status.
+     * @return AuditLogEntry - an initialized audit log entry.
+     */
+    protected static AuditLogEntry populateBundleAuditLogEntry(AuditLogEntry entry, Bundle.Entry responseEntry, HttpServletRequest request, Resource resource,
+        Date startTime, Date endTime, Response.Status responseStatus) { 
```

Comment:
minor formatting:  for method signatures that span multiple lines we double-indent to offset from the first line of the implementation
```suggestion
            Date startTime, Date endTime, Response.Status responseStatus) { 
```

Reply:
This has been fixed.